lib/gateway/Shard.js

```diff
@@ -549,7 +552,7 @@ class Shard extends EventEmitter {
                 */
                 this.emit("messageReactionAdd", message || {
                     id: packet.d.message_id,
-                    channel: channel || { id: packet.d.channel_id }
+                    channel: channel || {id: packet.d.channel_id}
```

Comment:
might not want to fix lint errors from changes that are not directly yours in this PR I guess 🤷‍♂️

Reply:
Sorry i just did fix all with eslint popup. I think once the other lint prs are merged this will automatically be removed